src/main/java/com/blackducksoftware/ohcount4j/detect/AbstractExtnResolver.java

```diff
@@ -32,11 +32,18 @@ protected String headContent(SourceFile sourceFile) throws IOException {
     }
 
     protected CharSequence getCharSequence(SourceFile sourceFile) throws IOException {
-        return sourceFile.getCharSequence();
+        CharSequence charSequence = sourceFile.getCharSequence();
```

Comment:
Similar note to that below.  `sourceFile.getCharSequence` should *always* return a valid CharSequence (either populated or empty) so that the methods that use it don't have to check its return value.